src/client/sandbox/code-instrumentation/methods.ts

```diff
@@ -77,21 +77,21 @@ export default class MethodCallInstrumentation extends SandboxBase {
                 if (isNullOrUndefined(owner) && !optional)
```

Comment:
I believe that we need to add null reference check here. Smth like follows:
```JS
if (isNullOrUndefined(owner)) {
    if (optional)
        return void 0;

    MethodCallInstrumentation._error(`Cannot call method '${methName}' of ${inaccessibleTypeToStr(owner)}`);
}
```

Reply:
@Aleksey28 please check if proposed solution is appropriate

Reply:
Yes, that will be better 👍